src/Microsoft.TemplateEngine.Core.Contracts/IOperation.cs

```diff
@@ -2,10 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Collections.Generic;
-using System.IO;
 
 namespace Microsoft.TemplateEngine.Core.Contracts
 {
+    public delegate void StreamWriteAction(byte[] buffer, int offset, int count);
```

Comment:
what is a background of this change?
`HandleMatch` can do other things with target\source streams via `processor`.

Reply:
I wanted to limit the exposure of internals and a scope of what can be done with targed stream.
Though it's a good point that it would be better to add the method directly into `IProcessorState` - adjusting...